site/class-1-load/Senmetsu/index.html

```diff
@@ -0,0 +1,60 @@
+<!doctype html>
+<meta charset=utf8>
+<!--Update the title with your username:-->
+<title>@Senmetsu</title>
+<meta content=width=device-width,initial-scale=1 name=viewport>
+<link href="https://fonts.googleapis.com/css?family=Roboto+Slab" rel="stylesheet">
+<link rel="stylesheet" href="css/style.css">
+
+<!--Link to d3 here:-->
```

Comment:
Remove the comments from your HTML. Tip. Here in your PR, on this file `index.html`, click the pencil icon in the top right to edit it in on GitHub.